specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/jobs.json

```diff
@@ -0,0 +1,814 @@
+          "readOnly": true
+        },
+        "message": {
+          "description": "A human-readable representation of the message code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdTimeUtc": {
+          "format": "date-time",
+          "description": "Time in UTC at which the message was created.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "LabelingJobProperties": {
+      "description": "Definition of a labeling job.",
+      "type": "object",
+      "properties": {
+        "labelCategories": {
+          "description": "Label categories of the job.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/LabelCategory"
+          },
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "jobInstructions": {
+          "description": "Instructions for the job.",
+          "$ref": "#/definitions/LabelingJobInstructions",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "datasetConfiguration": {
+          "description": "Dataset configuration for the job.",
+          "$ref": "#/definitions/LabelingDatasetConfiguration",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "mlAssistConfiguration": {
+          "description": "Machine learning assisted configuration for the job.",
+          "$ref": "#/definitions/MLAssistConfiguration",
+          "x-ms-mutability": [
+            "create",
+            "read",
+            "update"
+          ]
+        },
+        "labelingJobMediaProperties": {
+          "description": "Media specific properties in a labeling job.",
+          "$ref": "#/definitions/LabelingJobImageProperties",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "projectId": {
+          "format": "uuid",
+          "description": "Internal id of the job(Previously called project).",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "description": "Status of the job.",
+          "$ref": "#/definitions/JobStatus",
+          "readOnly": true
+        },
+        "progressMetrics": {
+          "description": "Progress metrics of the job.",
+          "$ref": "#/definitions/ProgressMetrics",
+          "readOnly": true
+        },
+        "statusMessages": {
+          "description": "Status messages of the job.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StatusMessage"
+          },
+          "readOnly": true
+        },
+        "createdTimeUtc": {
+          "format": "date-time",
+          "description": "Created time of the job in UTC timezone.",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
```

Comment:
Don't reuse top level property names.  If you need an internal "id" field name it something like "jobId".  "id" is reserved for the top level resource id of the job

Reply:
Removed `Id`, the resource can be identified using the top level `Id` property.